src/portal.py

```diff
@@ -185,6 +192,7 @@ def portal_me():
 
 def portal_update(
     fullname: Union[str, None] = None,
+    mainemail: Union[str, None] = None,
```

Comment:
From https://github.com/YunoHost/yunohost-portal/pull/21 it looks like the data is called `mail` and not `mainemail` so uuuuuh did you actually try the code that change the mainemail or am i missing something x_X

(In fact i would rather stick to calling this arg `mail` to be somewhat consistent with the admin API which also use `mail` for the main email arg in `user_update()`)

Reply:
Well, euh maybe at the time when I did this I didn't get that theses fields was mapped. From what I remember it was working so it's strange that it was working. Anyway I did the fix and tested, now it's working so it should be ok.